ajax/libs/jquery.adaptive-backgrounds/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "author": "Brian Gonzalez",
+  "name": "jquery.adaptive-backgrounds",
+  "description": "A simple jQuery plugin to extract the dominant color of an image and apply it to the background of its parent element.",
+  "version": "1.0.3",
+  "repository": {
+    "type": "git",
+    "url": "http://briangonzalez.github.io/jquery.adaptive-backgrounds.js/"
+  },
+  "filename": "jquery.adaptive-backgrounds.min.js",
+  "homepage": "https://briangonzalez.github.io/jquery.adaptive-backgrounds.js/",
+  "license": "MIT",
+  "keywords": [
+    "jquery",
+    "adaptive",
+    "background"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/briangonzalez/jquery.adaptive-backgrounds.js.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "jquery.adaptive-backgrounds*"
+        ]
+      },
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
So, let's remove `"basePath": "dist",` part.
What do you think?

Thank you very much.